src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -143,6 +149,15 @@ private void parse(String [] argv) {
 		if ( options.has("kafka_topic"))
 			this.kafkaTopic = (String) options.valueOf("kafka_topic");
 
+		if ( options.has("kafka_partition_by"))
+			this.kafkaPartitionKey = (String) options.valueOf("kafka_partition_by");
+
+		if ( options.has("kafka_partition_hash"))
+			this.kafkaPartitionHash = (String) options.valueOf("kafka_partition_hash");
+
+		if ( options.has("kafka_murmur3_seed"))
+			this.murmur3Seed = Integer.valueOf((String) options.valueOf("kafka_murmur3_seed"));
```

Comment:
is there a rationale for making this user-configurable?  I could definitely see documenting the seed value so you could reproduce it elsewhere, not totally sure I see the need to configure it though....